src/containers/UserInfo.tsx

```diff
@@ -0,0 +1,117 @@
+import { Card, Button, Row } from "react-bootstrap";
+import { GeoAlt, Link45deg, Balloon, Calendar3 } from "react-bootstrap-icons";
+import styled from "styled-components";
+
+const StyledCardBody = styled(Card.Body)`
+  height: 0px;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+`;
+
+interface UserInfoInterface {
+  headLogo: string;
+  userLogo: string;
+  username: string;
+  flagLogo: string;
+  userNickname: string;
+  aboutMeText: string;
+  userLocation: string;
+  userEmail: string;
+  userBirthday: string;
+  registrationDate: string;
+  inReadersCount: number;
+  readersCount: number;
+}
```

Comment:
I think some of these fields will not always be present. 
I have a suggestion to mark some of them "?". 
it means that this field is optional. Sometimes we do not show one of this field. For example email can be hidden from other users, but can be showen for you. 
Or userLogo may be missing